### DISCUSSION TITLE
Possible to show iLvl of vault rewards?

### DISCUSSION THREAD

**Discussion by Thann-K** [SavedInstances/SavedInstances#907] [Ideas]
Not sure if this is feasible or even possible, but I would love to be able to use SavedInstances to track which chars have what iLvl of reward in the vault slots without having to log from char to char to find out. Thanks.

**Comment by LiangYuxuan** [comment]
https://github.com/SavedInstances/SavedInstances/issues/939#issuecomment-2645809905
